vj4/server.py

```diff
@@ -13,7 +13,7 @@
 from vj4 import app
 from vj4.util import options
 
-options.define('listen', default='http://127.0.0.1:8888', help='Server listening address.')
+options.define('listen', default='http://0.0.0.0:8888', help='Server listening address.')
```

Comment:
revert this line plz, you can use `--listen` to specify the address you want to listen on